tests/test_endpoints/test_event.py

```diff
@@ -83,7 +83,7 @@ def test_update_event(client):
     assert response.status_code == 200
 
     event = db.events.find_one({'eid': eid})
-    assert event["title"] == update_field["title"]
+    assert event and event["title"] == update_field["title"]
```

Comment:
Not sure, but i think the test will fail if the event doesn't exist :thinking:

Reply:
Yes maybe, but pyright complains about event may be None, and then not being subscriptable, so to get rid of the error from pyright we can just assert than event is not None

Reply:
I see, nice to check anyways :+1: